support-frontend/app/views/main.scala.html

```diff
@@ -30,6 +30,8 @@
       </script>
     }
 
+    <script defer id="stripe-js" src="https://js.stripe.com/v3/"></script>
```

Comment:
This script tag already exists on line 148 of this file

Reply:
Ah yes... I think Tom Forbes and I may have been making changes at the same time